android/build.gradle

```diff
@@ -80,7 +80,7 @@ android {
     compileSdkVersion safeExtGet('compileSdkVersion', 31)
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 21)
-        targetSdkVersion safeExtGet('targetSdkVersion', 29)
+        targetSdkVersion safeExtGet('targetSdkVersion', 31)
```

Comment:
Have you tested that this works? We had to revert back to 29 due to a crash on startup: https://github.com/microsoft/react-native-test-app/commit/f87322599c8c0d4b0a510616ae1caf5b5f4978c4

Reply:
That's interesting - why you would have a crash on startup due to targetSdk? It's used as compile time and by IDE (to give you latest warnings/deprecations etc)

Reply:
Ah, it seems it was a bug in okHttp parsing android version 😄 
https://stackoverflow.com/a/64235713

Reply:
Fixed here
https://square.github.io/okhttp/changelogs/changelog_3x/#version-3147